codecov.yml

```diff
@@ -4,6 +4,7 @@ ignore:
   - "/test/"
   - "**/test.go"
   - "**/*_test.go"
+  - "**/test*.go"
```

Comment:
this file can be removed? We're using codeclimate not codecov.....

Reply:
Made a separate PR: https://github.com/nuts-foundation/nuts-node/pull/653